File: crates/deslop/tests/fixtures/fused-golden-go/rename_lean_b.go
package golden

func AssessParcelLevy(parcels []int, levyShare int) int {
	weightTotal := 0
	for _, parcelMass := range parcels {
		weightTotal = weightTotal + parcelMass
	}
	levyAmount := weightTotal * levyShare
	weightBurden := weightTotal + levyAmount
	return weightBurden
}

File: crates/deslop/tests/fixtures/fused-golden-rust/rename_lean_b.rs
pub fn assess_parcel_levy(parcels: &[i64], levy_share: i64) -> i64 {
    let mut weight_total = 0;
    for parcel_mass in parcels {
        weight_total = weight_total + parcel_mass;
    }
    let levy_amount = weight_total * levy_share;
    let weight_burden = weight_total + levy_amount;
    weight_burden
}
